src/main/resources/templates/start.html

```diff
@@ -54,7 +54,7 @@ <h4> A restaurant is a place where you can enjoy the food you like with your fam
 
             <button class="btn btn-info"> Create Your Order </button>
 
-             <p3> <br>Don't have an account with us? Sign Up </p3>
+             <p3> <br>Don't have an account with us? <a href="#signUp">Sign Up</a> </p3>
```

Comment:
Shouldn't a # href point to an a tag?

Reply:
When we # it points to a corresponding id on the page. So it just jumps down the page to the already created div. It could point to an a tag. Is it better practice to do this?